docs/admin/authorization.md

```diff
@@ -167,6 +169,85 @@ For example, if you wanted to grant the default service account in the kube-syst
 
 The apiserver will need to be restarted to pickup the new policy lines.
 
+## Webhook Mode
+
+When specified, mode `Webhook` causes Kubernetes to query an outside REST service when determining user privileges.
+
+### Configuration File Format
+
+Mode `Webhook` requires a file for HTTP configuration, specify by the `--authorization-webhook-config-file=SOME_FILENAME` flag.
+
+The configuration file uses the [kubeconfig](../user-guide/kubeconfig-file.md) file format. Within the file "users" refers to the API Server webhook and "clusters" refers to the remote service.
+
+A configuration example which uses HTTPS client auth:
+
+```yaml
+# clusters refers to the remote service.
+clusters:
+  - name: name-of-remote-authz-service
+    cluster:
+      certificate-authority: /path/to/ca.pem      # CA for verifying the remote service.
+      server: https://authz.example.com/authorize # URL of remote service to query. Must use 'https'.
+
+# users refers to the API Server's webhook configuration.
+users:
+  - name: name-of-api-server
+    user:
+      client-certificate: /path/to/cert.pem # cert for the webhook plugin to use
+      client-key: /path/to/key.pem          # key matching the cert
+```
+
+### Request Payloads
+
+When faced with an authorization decision, the API Server POSTs a JSON serialized api.authorization.v1beta1.SubjectAccessReview object describing the action. This object contains fields describing the user attempting to make the request, and either details about the resource being accessed or requests attributes. An example request body:
+
+```json
+{
+  "spec": {
+    "resourceAttributes": {
+      "namespace": "kittensandponies",
+      "verb": "GET",
+      "group": "*",
+      "resource": "pods"
+    },
+    "user": "jane",
+    "group": [
+      "group1",
+      "group2"
+    ]
+  }
+}
+```
+
+The remote service is expected to fill the SubjectAccessReviewStatus field of the request and respond to either allow or disallow access. A permissive response would return:
+
+```json
+{
+  "spec": {
+    ...
```

Comment:
I think it will (or should) just omit the "spec" field entirely.  Can you confirm and fix docs?